api/src/main/java/uk/gov/dhsc/htbhf/claimant/communications/PaymentCycleNotificationHandler.java

```diff
@@ -54,11 +58,22 @@ private boolean voucherEntitlementIndicatesNewChildFromPregnancy(PaymentCycle pa
 
     private void handleUpcomingBirthdayEmails(PaymentCycle paymentCycle) {
         NextPaymentCycleSummary nextPaymentCycleSummary = childDateOfBirthCalculator.getNextPaymentCycleSummary(paymentCycle);
-        if (nextPaymentCycleSummary.hasChildrenTurningFour()) {
-            upcomingBirthdayEmailHandler.sendChildTurnsFourEmail(paymentCycle, nextPaymentCycleSummary);
-        }
-        if (nextPaymentCycleSummary.hasChildrenTurningOne()) {
-            upcomingBirthdayEmailHandler.sendChildTurnsOneEmail(paymentCycle, nextPaymentCycleSummary);
+        boolean isNotPregnant = isNotPregnant(paymentCycle);
+        if (nextPaymentCycleSummary.youngestChildTurnsFour() && isNotPregnant) {
+            upcomingBirthdayEmailHandler.sendPaymentStoppingYoungestChildTurnsFourEmail(paymentCycle, nextPaymentCycleSummary);
+        } else {
+            if (nextPaymentCycleSummary.hasChildrenTurningFour()) {
+                upcomingBirthdayEmailHandler.sendChildTurnsFourEmail(paymentCycle, nextPaymentCycleSummary);
+            }
+            if (nextPaymentCycleSummary.hasChildrenTurningOne()) {
+                upcomingBirthdayEmailHandler.sendChildTurnsOneEmail(paymentCycle, nextPaymentCycleSummary);
+            }
         }
     }
-}
+
+    private boolean isNotPregnant(PaymentCycle paymentCycle) {
```

Comment:
I think this should go in the `PregnancyEntitlementCalculator` and could have a better name than this, something like `isNotPregnantAfterCycle`.

Reply:
Good point done - but the method is `claimantIsPregnantAfterCycle` to be in keeping with the existing `claimantIsPregnantInCycle`